### PR TITLE
Automatically install Python dependencies using autopy

### PR DESCRIPTION
**Pull request by baltpeter**

### CODE REVIEW

File: src/android.ts [hunk 1]
```diff
@@ -528,10 +532,10 @@ export const androidApi = <RunTarget extends SupportedRunTarget<'android'>>(
                 throw new Error(`Invalid battery optimization state: ${state}`);
         }
     },
-    startApp(appId) {
+    async startApp(appId) {
         // We deliberately don't await these since objection doesn't exit after the app is started.
         if (options.capabilities.includes('certificate-pinning-bypass')) {
-            const process = execa('objection', [
+            const process = (await venv)('objection', [
```
Comment (zner0L) [inline src/android.ts:538]: Why is it `venv` and not `python`?

**Review by zner0L — CHANGES_REQUESTED** [review]
I like that a lot. This will make everything so much simpler.

File: src/android.ts (hunk 1 above)
Comment (baltpeter) [reply src/android.ts:538]: We don't want to await objection, but `python`, we can only await the whole thing.

Comment (zner0L) [reply src/android.ts:538]: I see. A comment to clarify that would be nice.

File: package.json [hunk 2]
```diff
@@ -54,6 +54,7 @@
     "dependencies": {
         "@napi-rs/lzma": "^1.1.2",
         "andromatic": "^1.0.0",
+        "autopy": "/home/benni/coding/JS/tweasel/autopy/autopy-v0.0.0.tgz",
```
Comment (zner0L) [inline package.json:57]:
```suggestion
        "autopy": "1.0.0",
```

**Review by zner0L — APPROVED** [review]

**Pull request merged**